.envrc

```diff
@@ -5,6 +5,16 @@
 # If you'd like to override or set any custom environment variables, this .envrc will read a .env file at the end.
 set -e
 
+# Upgrading Mac can uninstall the Command Line Tools, thus, removing our access to git
+# The message talks about xcrun, however, we can use the lack of git as a way to know that we need this
+# xcrun: error: invalid active developer path (/Library/Developer/CommandLineTools),
+# missing xcrun at: /Library/Developer/CommandLineTools/usr/bin/xcrun
+if [ "$(uname -s)" == "Darwin" ] && [ ! -f "/Library/Developer/CommandLineTools/usr/bin/git" ]; then
+    echo -e "$(tput setaf 1)\nERROR: Complete the interactive installation (10+ mins) of the Command Line Tools.$(tput sgr0)"
```

Comment:
Ah I see, running xcode-select will open a gui for installing CLT...

what about

`ERROR: xcode Command Line Tools missing... we are running `xcode-select --install` for you, please complete installation (10+ mins)`

Reply:
OK. Next time I pass by the code I will improve the wording. Thanks for the review!